### PR TITLE
Add db client for languages

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '3.1'
```
Comment (kabaros) [inline docker-compose.yml]: Add to README how to use docker if you want to add it (and show it to people too)

File: server/app.js [hunk 2]
```diff
@@ -80,8 +81,9 @@ app.use(function (req, res, next) {
 app.use('/', index)
 // .use('/admin/articles', ensureAuthenticated, adminArticles)
 //  app.use('/admin/categories', ensureAuthenticated, adminCategories)
-app.use('/api', router)
-app.use('/users', autentication)
+//app.use('/api', router)
+app.use('/languages', language)
```
Comment (kabaros) [inline server/app.js]: this should be under /api (i.e. /api/languages)

File: server/__integration_tests/dbClients/languagesDB.spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+const request = require('supertest')
+const language = require('../../dbClients/languageDB')
+
+describe('CRUD /languages', () => {
+    it('should respond with 200', () => {
+        return language.getLanguages().then(response => {
```
Comment (kabaros) [inline server/__integration_tests/dbClients/languagesDB.spec.js]: how is this test passing??? getLanguages() should not be returns statusCode, it's not an api

File: server/__integration_tests/dbClients/languagesDB.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                "original_name": "عربي"
+            },
+            {
+                "language_id": "am",
+                "short_name": "am",
+                "long_name": "amharic",
+                "original_name": "amharic"
+            }
+        ]
+        return language.getLanguages().then(response => {
```
Comment (kabaros) [inline server/__integration_tests/dbClients/languagesDB.spec.js]: again .. how is this passing?

File: server/__integration_tests/api/languages.spec.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    })
+
+    it('should edit new languages', () => {
+        const data = {
+            language_id: 'en',
+            short_name: 'ar',
+            long_name: 'arabic',
+            original_name: 'arabic'
+        }
+        return request(language).put('/languages/edit', data).then(response => {
```
Comment (kabaros) [inline server/__integration_tests/api/languages.spec.js]: again, how is this passing? I don't see a /edit endpoint in code

File: server/__integration_tests/dbClients/languagesDB.spec.js (hunk 3 above)
Comment (DwinaTech) [reply server/__integration_tests/dbClients/languagesDB.spec.js]: Done

File: server/__integration_tests/dbClients/languagesDB.spec.js (hunk 4 above)
Comment (DwinaTech) [reply server/__integration_tests/dbClients/languagesDB.spec.js]: Done

File: docker-compose.yml (hunk 1 above)
Comment (DwinaTech) [reply docker-compose.yml]: Done

File: server/app.js (hunk 2 above)
Comment (DwinaTech) [reply server/app.js]: Done

**Pull request merged**